docs/source/services/timelock_service/installation.rst

```diff
@@ -0,0 +1,58 @@
+.. _timelock-installation:
+
+Timelock Server Installation
+============================
+
+1. Clone the git repository.
+
+   .. code:: bash
+
+      git clone git@github.com:palantir/atlasdb.git; cd atlasdb
+
+2. Generate a tar file with Gradle:
+
+   .. code:: bash
+
+      ./gradlew atlasdb-timelock-server:distTar
+
+  This will place a tar file in the ``build/distributions`` directory of the ``atlasdb-timelock-server`` project. The
+  tar file follows semantic versioning.
+
+3. Copy this tar file to the server on which you want to deploy Timelock Server.
+   (If you wish to run Timelock Server locally, this step may be omitted.)
+
+   .. code:: bash
+
+      scp atlasdb-timelock-server/build/distributions/atlasdb-timelock-server-0.25.0.sls.tgz <DESTINATION_HOST>
```

Comment:
We probably don't want to list `scp` commands, etc. We should basically say configure the timelock server by editing the file at `var/conf/timelock.yml` and you can start the timelock server with `service/bin/init.sh start`